api/graphql/schema.graphql

```diff
@@ -1515,8 +1519,13 @@ type Post {
   locationObject: Location
   # Logged in user's attendance response to an event post: 'yes', 'maybe' or 'no'
   myEventResponse: String
-  # Whether the logged in user has upvoted the post or not
+  # The reactions of the logged-in user to this
+  myReactions: [Reaction]
+  # Whether the logged-in user has upvoted the post or not
   myVote: Boolean
+  # Number of people that have one or more reactions to a post
+  peopleReactionsTotal: Int
```

Comment:
i would call this peopleReactedTotal, or even peopleWhoReacted , but again, not a big deal and I can be convinced